src/Components/App/App.js

```diff
@@ -1,19 +1,20 @@
 import React, { Component } from 'react';
-import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
 import Header from '../Header/Header';
 import Footer from '../Footer/Footer';
 import Movies from '../Movies/Movies';
-import Details from '../Details/Details'
-import About from '../About/About'
-import Contact from '../Contact/Contact'
-import { fetchAllMovies } from '../../apiCalls'
-import './App.css'
+import Details from '../Details/Details';
+import About from '../About/About';
+import Contact from '../Contact/Contact';
+import { fetchAllMovies } from '../../apiCalls';
+import './App.css';
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
       allMovies: [],
+      filteredMovies: null,
```

Comment:
Im happy we found this solution! Its still a little fuzzy for me about why so lets keep looking into it a bit!